File: Contar vocales de una palabra/Contar vocales de una palabra/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Contar_vocales_de_una_palabra
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            string Palabra; 
            int contador1 = 0, contador2 = 0;

            Palabra = Convert.ToString(txtpalabra.Text);

            for (int i = 0; i < Palabra.Length; i++)
            {
                contador1++;
            }
            for (int i = 0; i < Palabra.Length; i++)
            {
                if ((Palabra[i] == 'a') || (Palabra[i] == 'e') || (Palabra[i] == 'i') || (Palabra[i] == 'o') || (Palabra[i] == 'u'))
                {
                    contador2++;

                }
            }
            MessageBox.Show("La palabra " + Palabra + " tiene " + contador2 + " Vocales.");
            
        }
        
        }
    }

File: resolución del Teorema de Pitágoras/resolución del Teorema de Pitágoras/Form1.vb
Public Class Form1

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim a As Integer
        Dim b As Integer
        Dim c As Integer
        Dim r As Integer
        a = txt1.Text * txt1.Text
        b = txt2.Text * txt2.Text
        c = a + b
        r = c * c
        MessageBox.Show("El resultado es: " & r.ToString)
    End Sub
End Class
